projector/binding.go

```diff
@@ -57,6 +58,11 @@ func (p *serviceBindingProjector) Project(ctx context.Context, binding *serviceb
 	if err != nil {
 		return err
 	}
+	secret, err := p.bindingSecrets(binding, workload)
+	if err != nil {
+		return err
+	}
+	binding.Status.Binding = &servicebindingv1beta1.ServiceBindingSecretReference{Name: secret}
```

Comment:
what is this trying to do? By default it appears to be a no-op looking up `binding.Status.Binding.Name` and setting that same value back. It's find for the projector to err if that value is not set.

Reply:
The intent here was to set `.status.binding.name` in the service binding to point at the secret containing binding information.      In hindsight, this is most incorrect - I'm not even pulling the secret name in from the service.  Fixed.